msal/src/main/java/com/microsoft/identity/client/PublicClientApplicationConfiguration.java

```diff
@@ -386,12 +414,13 @@ void mergeConfiguration(PublicClientApplicationConfiguration config) {
         } else if (config.mBrowserSafeList != null) {
             this.mBrowserSafeList.addAll(config.mBrowserSafeList);
         }
-
         // Multiple is the default mode.
         this.mAccountMode = config.mAccountMode != AccountMode.MULTIPLE ? config.mAccountMode : this.mAccountMode;
         this.mClientCapabilities = config.mClientCapabilities == null ? this.mClientCapabilities : config.mClientCapabilities;
         this.mIsSharedDevice = config.mIsSharedDevice == true ? this.mIsSharedDevice : config.mIsSharedDevice;
         this.mLoggerConfiguration = config.mLoggerConfiguration == null ? this.mLoggerConfiguration : config.mLoggerConfiguration;
+        this.webViewZoomControlsEnabled = config.webViewZoomControlsEnabled;
```

Comment:
We need to check here for null like we are doing everywhere else to see if the values of zoom controls and zoom enabled in the developer provided config are null, then we need to use the values from the default config.

@arislomovic You also need to specify a default value of `true` for these controls in the `msal_default_config.json`. See this file: https://github.com/AzureAD/microsoft-authentication-library-for-android/blob/dev/msal/src/main/res/raw/msal_default_config.json

so that when the developer doesn't specify a value for these fields in their config, then we can use the default `true` from the msal default config

Reply:
@shahzaibj  I changed it. I think that should do it. Take a look at the files, though, as I squashed the commits

Reply:
Looks good to me. I've approved it. Thanks!

Reply:
I'll go ahead and merge this into our dev, and then I'll open another PR to update the submodule